### PR TITLE
Fix NPD batch merging.

### PR DESCRIPTION
**Pull request by Random-Liu**
Use `batch` for batch merging.

### CODE REVIEW

File: config/jobs/kubernetes/node-problem-detector/node-problem-detector-presubmits.yaml [hunk 1]
```diff
@@ -79,7 +79,7 @@ presubmits:
         - bash
         - -c
         - >-
-          ./test/build.sh -p $(PULL_NUMBER) pr &&
+          ./test/build.sh -p ${PULL_NUMBER:-batch} pr &&
```
Comment (BenTheElder) [inline config/jobs/kubernetes/node-problem-detector/node-problem-detector-presubmits.yaml:82]: I think this is only safe if we make the (bad) assumption that there's only one batch at a time ever.
https://github.com/kubernetes/node-problem-detector/blob/35bfe697a5fd9b058f86a3fb704b7e6d7b68457c/test/build.sh#L130

Maybe e.g. BUILD_ID

Comment (BenTheElder) [reply config/jobs/kubernetes/node-problem-detector/node-problem-detector-presubmits.yaml:82]:
```suggestion
          ./test/build.sh -p ${BUILD_ID} pr &&
```

Comment (Random-Liu) [reply config/jobs/kubernetes/node-problem-detector/node-problem-detector-presubmits.yaml]: Done

Comment (Random-Liu) [reply config/jobs/kubernetes/node-problem-detector/node-problem-detector-presubmits.yaml]: Thanks for the suggestion!

**Comment by wangzhen127** [comment]
Probably update the PR description.

/lgtm
/hold

Unhold when the NPD side change is ready.

**Comment by Random-Liu** [comment]
@wangzhen127 We don't need to change NPD side. This should just work. :)

The NPD side is more a cleanup or an improvement.

**Comment by wangzhen127** [comment]
Ah, I see.

/unhold

**Pull request merged**